starter/js/atm.js

```diff
@@ -1,42 +1,32 @@
-//Begin with the document ready function
-
-    //Checking account deposit function
-
-      //On click of the depositChecking button
-
-        //Get value from the amountChecking input field
-
-        //Take that value and add it to the existing value in the checkingBalance div
-
-    //Checking account withdrawl funtion
-
-      //On click of the withdrawChecking button
-
-        //Get value from the amountChecking input field
-
-        // If that value is greater than the value in the account ignore that action
-        // In other words if this would put the account into a negative balance do not allow it
-
-        //Else subtract that value from the current amount in the checking account
-
-    //Savings account deposit function
-
-      //On click of the depositSavings button
-
-        //Get value from the amountSavings input field
-
-        //Take that value and add it to the existing value in the savingsBalance div
-
-    //Savings account withdraw funtion
-
-      //On click of the withdrawl button
-
-        //Get value from the amountSavings input field
-
-         //If that value is greater than the value in the account ignore that action
-         //In other words if this would put the account into a negative balance do not allow it
-
-         //Else subtract that value from the current amount in the savings account
+$(document).ready(function() {
+  $('#depositChecking').on('click', function() {
+    return ('#amountChecking').val();
+    return amountChecking + checkingBalance
+  })
+  $('#withdrawChecking').on('click', function() {
+    return amountChecking.val();
+    if ('#withdrawChecking' < 1) {
+      console.log('No chance you are taking any money out mate');
+    }
+    else {
+      return withdrawChecking - amountChecking
+    }
+  })
+
+  $('#depositSavings').on('click', function() {
+    return amountSavings.val();
+    return amountSavings + savingsBalance
```

Comment:
again only can `return` one thing, but in this case, we don't want to return at all, we want to put the new value into the dom.